File: data/museum/oseg_generated/ruby/BuyMuseumTickets_event_entry.rb
require "openapimuseum_client"

OpenapiMuseumClient.configure do |config|
end

buy_museum_tickets = OpenapiMuseumClient::BuyMuseumTickets.new
buy_museum_tickets.ticket_type = "event"
buy_museum_tickets.event_id = "dad4bce8-f5cb-4078-a211-995864315e39"
buy_museum_tickets.ticket_date = "2023-09-05"
buy_museum_tickets.email = "todd@example.com"
buy_museum_tickets.ticket_id = nil

begin
    api_caller = OpenapiMuseumClient::TicketsApi.new

    response = api_caller.buy_museum_tickets(
        buy_museum_tickets,
    )

    p response
rescue OpenapiMuseumClient::ApiError => e
    puts "Exception when calling Tickets#buy_museum_tickets: #{e}"
end

File: data/museum/oseg_generated/csharp/BuyMuseumTickets_event_entry.cs
using System;
using System.Collections.Generic;
using System.IO;

using Org.OpenAPIMuseum.Api;
using Org.OpenAPIMuseum.Client;
using Org.OpenAPIMuseum.Model;

public class BuyMuseumTicketsEventEntry
{
    public static void Main()
    {
        var config = new Configuration();

        var buyMuseumTickets = new BuyMuseumTickets(
            email: "todd@example.com",
            ticketId: null,
            ticketDate: "2023-09-05",
            ticketType: BuyMuseumTickets.TicketTypeEnum.Event,
            eventId: "dad4bce8-f5cb-4078-a211-995864315e39"
        );

        try
        {
            var apiCaller = new TicketsApi(config);

            var response = apiCaller.BuyMuseumTickets(
                buyMuseumTickets: buyMuseumTickets
            );

            Console.WriteLine(response);
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling Tickets#BuyMuseumTickets: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}
